tinycolor/resources/deps.cljs

```diff
@@ -0,0 +1,5 @@
+{:foreign-libs
```

Comment:
This file shouldn't exist when using `deps-cljs` task, which will generate the file.

Reply:
Makes sense, I will update both PRs. Would you like me to rebase or will you squash on commit?

Reply:
Multiple commits are fine. I have stopped caring too much about best git commit/merge practices with this repo :)